### ISSUE TITLE
Could you plz offer me the data-split files used in the experiments on Pascal Context?

### ISSUE THREAD

**Issue by revaeb** [yh-pengtu/GuidedMix-Net#2]
Congratulations for your excellent work!
I want to commit semi-supervised experiments on Pascal Context dataset and compare the results with your work. So could you plz offer me the data-split files used in the experiments on Pascal Context? Thank you so much! :)

**Comment by yh-pengtu** [comment]
Considering the fairness of the experimental comparison, the data list of all experiments is not fixed.
The experimental results under each label-ratios are averaging the mAP from three randomly split data lists.

Hope that helps!

**Comment by revaeb** [comment]
> Considering the fairness of the experimental comparison, the data list of all experiments is not fixed. The experimental results under each label-ratios are averaging the mAP from three randomly split data lists.
> 
> Hope that helps!

Thx for your reply！
So you just split the whole training set into different labeled ratio like 1/8 labeled images and 7/8 unlabeled images and test the model on the whole validation set?

**Comment by yh-pengtu** [comment]
yeap!

revaeb ***@***.***> 于2022年3月21日周一 15:52写道：
